go.mod

```diff
@@ -23,7 +23,6 @@ require (
 	github.com/opencontainers/runc v1.1.2
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210910115017-0d6cc581aeea
 	github.com/pelletier/go-toml v1.9.3
-	github.com/pkg/errors v0.9.1
```

Comment:
@kzys how are `go.sum` changes propagated?

Reply:
`go mod tidy` but if any of our transitive dependencies include it, then it will still be there.

Reply:
Ah that was my confusion.